models/brac/function/get_dashboard_data_anc_impact.sql

```diff
@@ -0,0 +1,522 @@
+    
+    place_period.period_start AS _period_start,
+    date_part('epoch',place_period.period_start)::numeric AS _period_start_epoch,
+    
+    CASE
+        WHEN param_facility_group_by = 'health_center'
+            THEN place_period.health_center_uuid
+        WHEN param_facility_group_by = 'district_hospital'
+            THEN place_period.district_hospital_uuid
+        ELSE
+            'All'
+    END AS _facility_join_field,
+        
+        /* By Reg Date */
+        SUM(COALESCE(count_new_person_by_reg,0)) AS count_new_person_by_reg,
+        SUM(COALESCE(count_preg_by_reg,0)) AS count_preg_by_reg,
+        SUM(COALESCE(count_preg_with_lmp_by_reg,0)) AS count_preg_with_lmp_by_reg,
+        SUM(COALESCE(count_preg_without_lmp_by_reg,0)) AS count_preg_without_lmp_by_reg,
+        
+        
+        SUM(COALESCE(count_preg_early_reg_by_reg,0)) AS count_preg_early_reg_by_reg,
+        SUM(COALESCE(count_deliv_by_reg,0)) AS count_deliv_by_reg,
+        SUM(COALESCE(count_danger_sign_by_reg,0)) AS count_danger_sign_by_reg,
+        SUM(COALESCE(count_pregnancy_visit_by_reg,0)) AS count_pregnancy_visit_by_reg,
+    
+        CASE
+            WHEN SUM(COALESCE(count_preg_by_reg,0)) = 0
+            THEN 0::float
+            ELSE (({{ ref("safe_divide") }}(SUM(COALESCE(count_preg_early_reg_by_reg,0)), SUM(COALESCE(count_preg_by_reg,0))))* 100)            
+        END AS percent_preg_early_reg_by_reg,
+        
+        /* By MDD */
+        
+        SUM(COALESCE(count_preg_by_mdd,0)) AS count_preg_by_mdd,
+        SUM(COALESCE(count_preg_with_lmp_by_mdd,0)) AS count_preg_with_lmp_by_mdd,
+        SUM(COALESCE(count_preg_without_lmp_by_mdd,0)) AS count_preg_without_lmp_by_mdd,
+        SUM(COALESCE(count_preg_confirmed_deliv_by_mdd,0)) AS count_preg_confirmed_deliv_by_mdd,
+        SUM(COALESCE(count_preg_1plus_visit_by_mdd,0)) AS count_preg_1plus_visit_by_mdd,    
+        SUM(COALESCE(count_preg_4plus_visit_by_mdd,0)) AS count_preg_4plus_visit_by_mdd,    
+        SUM(COALESCE(count_preg_early_reg_by_mdd,0)) AS count_preg_early_reg_by_mdd,
+        SUM(COALESCE(count_preg_early_reg_4plus_visit_by_mdd,0)) AS count_preg_early_reg_4plus_visit_by_mdd,
+        SUM(COALESCE(count_preg_danger_sign_by_mdd,0)) AS count_preg_danger_sign_by_mdd,
+        SUM(COALESCE(count_preg_danger_sign_4plus_visit_by_mdd,0)) AS count_preg_danger_sign_4plus_visit_by_mdd,    
+        
+        CASE
+            WHEN SUM(COALESCE(count_preg_by_mdd,0)) = 0
+            THEN 0::float
+            ELSE ({{ ref("safe_divide") }}(SUM(COALESCE(count_preg_confirmed_deliv_by_mdd,0)), SUM(COALESCE(count_preg_by_mdd,0))) * 100)          
+        END AS percent_deliv_confirmed_by_mdd,
+    
+        CASE
+            WHEN SUM(COALESCE(count_preg_by_mdd,0)) = 0
+            THEN 0::float
+            ELSE ({{ ref("safe_divide") }}(SUM(COALESCE(count_preg_1plus_visit_by_mdd,0)) / SUM(COALESCE(count_preg_by_mdd,0))) * 100)          
```

Comment:
This (and subsequent) expression(s) looks incorrect. We are passing a fraction to safe_divide. Look at the '/' vs having a comma.